scrapli_community/alcatel/aos/alcatel_aos.py

```diff
@@ -0,0 +1,17 @@
+"""scrapli_community.alcatel.aos.alcatel_aos"""
+
+from scrapli_community.alcatel.aos._async import default_async_on_close
+from scrapli_community.alcatel.aos.sync import default_sync_on_close
+
+SCRAPLI_PLATFORM = {
+    "driver_type": "generic",
+    "defaults": {
+        # Prompt can be anything, but best practice is to end with > or #
+        "comms_prompt_pattern": r"^[a-zA-Z0-9.\-_@\/:]{1,63}[>|#]\s*",
+        "sync_on_open": None,
```

Comment:
I think from our slack convo I remember seeing something similar to `term len 0` is that not generally needed?

Reply:
I tested both and term len 0 is not required, it was the missing "^" that caused the issue :)